### ISSUE TITLE
Improve confirmation page

### ISSUE THREAD

**Issue by kelliewellingtons** [barnardos/consent-form-builder#314]
_AS a user,
I NEED information on the final page of the process to be written in plain English,
SO that I can be confident that I have successfully completed the journey, and I am clear on what the next steps are._

This ticket includes:
- reordering the content (moving the preview above the SubmitAction)
- Removing links for only outputting the Information sheet/only outputting the Consent form
- Remove the detail on how to bookmark 

## Links and resources

Copy document:
https://docs.google.com/document/d/1g7ET6R6PlLQVHjgrP8dwM51vttOLnTgIQIkUm0aFsbA/edit?usp=sharing

## Acceptance Criteria

- [ ] Content and interactions to follow what is outlined in the linked document
- [ ] Works on target devices and browsers

**Comment by ritapatel7** [comment]
Estimated as 3 if still needs to be discussed with others and need content approved otherwise probably simple to update

**Comment by joelcarr** [comment]
The copy should be reviewed by @Jesskyri. We also should figure out a way to only have one version of the consent form to refer to the participant before doing these copy changes.

**Comment by joelcarr** [comment]
As talked about offline, we will have to do the appropriate research and testing into the idea of having one version of the consent form to the participant.
